Source/Core/Core/IOS/IOSC.h

```diff
@@ -17,6 +17,84 @@ class PointerWrap;
 
 namespace IOS
 {
+enum class SignatureType : u32
+{
+  RSA4096 = 0x00010000,
+  RSA2048 = 0x00010001,
+  // XXX: Add support for ECC (0x00010002).
+};
+
+enum class PublicKeyType : u32
+{
+  RSA4096 = 0,
+  RSA2048 = 1,
+};
+
+#pragma pack(push, 4)
+struct SignatureRSA4096
+{
+  SignatureType type;
+  u8 sig[0x200];
+  u8 fill[60];
+  char issuer[0x40];
```

Comment:
Maybe thats just in my head, but it feels a little weird that hex-literals are mixed with dec-literals. Especially further down where values like `64` and `0x40` are used right next to each other.
IMO at least those with `static_assert` should be consistent, to make it easier to verify.

Can be ignored if thats just me.